app/src/main/java/com/example/onestopgrocery/models/CartAndOrders.java

```diff
@@ -0,0 +1,33 @@
+package com.example.onestopgrocery.models;
+
+import androidx.room.Embedded;
+import androidx.room.Relation;
+
+import com.example.onestopgrocery.entities.Cart;
+import com.example.onestopgrocery.entities.Order;
+import com.example.onestopgrocery.entities.Product;
+import com.example.onestopgrocery.entities.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartAndOrders {
+    @Embedded
+    User user;
```

Comment:
This attribute should be public. Right now, they can't be accessed outside of the `CartAndOrders` class. The same should be true for `cart` and `orderList`.

Reply:
I have made them public. Work in Progress, I will update it more later, since now it has not been complete yet.